merobox/commands/manager.py

```diff
@@ -1535,13 +1551,18 @@ def _apply_e2e_defaults(
                 "sync.interval_ms": 500,
                 # 1s periodic checks (ensures rapid sync in tests)
                 "sync.frequency_ms": 1000,
-                # Ethereum local devnet configuration (same as e2e tests)
-                "context.config.ethereum.network": "sepolia",
+                # Ethereum local devnet configuration
+                "context.config.ethereum.network": "local",
                 "context.config.ethereum.contract_id": "0x5FbDB2315678afecb367f032d93F642f64180aa3",
                 "context.config.ethereum.signer": "self",
-                "context.config.signer.self.ethereum.sepolia.rpc_url": "http://127.0.0.1:8545",
-                "context.config.signer.self.ethereum.sepolia.account_id": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
-                "context.config.signer.self.ethereum.sepolia.secret_key": "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
+                "context.config.signer.self.ethereum.local.rpc_url": eth_rpc_url,
+                "context.config.signer.self.ethereum.local.account_id": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+                "context.config.signer.self.ethereum.local.secret_key": "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
+                # ICP local devnet configuration (for consistency)
+                "context.config.icp.network": "local",
+                "context.config.icp.contract_id": "bkyz2-fmaaa-aaaaa-qaaaq-cai",
+                "context.config.icp.signer": "self",
+                "context.config.signer.self.icp.local.rpc_url": icp_rpc_url,
```

Comment:
for all public/private keys, contract IDs, please define constants and then reuse them here.
Also, if you are using any keys like from Anvil, etc, please make sure to add a comment about it with a reference to where these keys are originally defined.